ocs_ci/ocs/resources/cloud_uls.py

```diff
@@ -48,6 +50,11 @@ def cloud_uls_factory(request, cld_mgr):
     except AttributeError:
         log.info("RGW is not available and was not initialized")
 
+    try:
+        ulsMap["aws-sts"] = cld_mgr.aws_sts_client
+    except Exception:
+        log.info("Cluster is not deployed STS mode")
```

Comment:
```suggestion
        log.warning("Cluster is not deployed STS mode")
```

Reply:
This is subjective... if its in non-sts cluster then it doesn't have to be warning. if its in sts cluster then yes.